packages/routing-utils/test/superstatic.spec.ts

```diff
@@ -938,7 +938,7 @@ test('convertRewrites', () => {
 test('convertHeaders', () => {
   const actual = convertHeaders([
     {
-      source: '(.*)+/(.*)\\.(eot|otf|ttf|ttc|woff|font\\.css)',
+      source: '(.*)/(.*)\\.(eot|otf|ttf|ttc|woff|font\\.css)',
```

Comment:
This fix (that @jeffsee55 tried before--not sure why it passes now) does pass the test suite.

This `+` doesn't make sense to me. I was trying to figure out if it was intentional. So, I checked with the original author of this test in case they remembered by this extra `+` was here, but they don't. They think it was a mistake.

I think this is a fine change for this PR. We'll have to discuss internally how we roll out this breaking change, though.